### DISCUSSION TITLE
怎么使用tun模式 ？

### DISCUSSION THREAD

**Discussion by kingvenlee** [v2rayA/v2rayA#1470] [General]
怎么使用tun模式 ？

我选了gvisor tun和system tun模式
两个都无法联网，
有没有相关文档，这两个模式怎么使用 ？
（我已经下载了sing-box放到了PATH）

**Comment by eronez** [comment]
Just select "*** tun" as transparent proxy mode.
If unable to connect to the network, please show corresponding v2raya log.
